nss-cohort-37/Workforce-Silver-Snakes
# Gdf computer controller

# Description
Created the ComputersController and all associated CRUD Views. Get all Computers, Editing of Computers, and deleting of  Computers. Also added Computers to the nav bar.

Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions

- Click on Computers in the nav bar
- Check to see Computer shows a list of all information (Id, PurchaseDate, Make, Model)
- Click Add Computer and Enter a PurchaseDate, Make, Model
- Save Computer, Check to see if information was added to the list
- Click Details on a Computer
- View details and see if the information is populated (Id, PurchaseDate, Make, Model)
- Go back and click Delete on a Computer and verify it works.

# Checklist:
